src/main/java/WayofTime/bloodmagic/util/handler/event/LivingArmourHandler.java

```diff
@@ -341,7 +344,7 @@ public static void onArrowFire(ArrowLooseEvent event)
                         ItemArrow itemarrow = (ItemArrow) ((stack.getItem() instanceof ItemArrow ? arrowStack.getItem() : Items.ARROW));
                         EntityArrow entityarrow = itemarrow.createArrow(world, arrowStack, player);
                         entityarrow.shoot(player, player.rotationPitch, player.rotationYaw, 0.0F, velocity * 3.0F, 1.0F);
-
+                        entityarrow.addTag("Arrow Shot");
```

Comment:
Unless vanilla uses tags named similarly, identifiers like this should be `snake_case`